server/routes/passwordReset.tsx

```diff
@@ -21,7 +21,11 @@ app.get(['/password-reset', '/password-reset/:resetHash/:slug'], (req, res, next
 			if (userData && userData.resetHash !== req.params.resetHash) {
 				hashIsValid = false;
 			}
-			if (userData && userData.resetHashExpiration < Date.now()) {
+			if (
+				userData &&
+				userData.resetHashExpiration &&
+				userData.resetHashExpiration < Number(new Date())
```

Comment:
Force cast date to number, `resetHashExpiration` is one of the few `Date`s that are typed as `number` instead of `string`